functions/send-email/2.3/function.json

```diff
@@ -0,0 +1,211 @@
+{
+  "description": "Send an e-mail using SMTP credentials v2.3",
+  "label": "Send Email",
+  "category": "External",
+  "icon": {
+    "color": "Orange",
+    "name": "AtIcon"
+  },
+  "options": [
+    {
+      "meta": {
+        "type": "Value",
+        "allowedKinds": ["STRING", "INTEGER"],
+        "validations": { "required": true }
+      },
+      "name": "host",
+      "label": "Hostname",
+      "info": "The hostname of the SMTP server you want to use",
+      "advanced": false,
+      "configuration": {
+        "placeholder": "smtp.flowmailer.net"
+      }
+    },
+    {
+      "meta": {
+        "type": "Value",
+        "allowedKinds": ["STRING"],
```

Comment:
Why not allow an INTEGER for the port as well? Now you only added that type for the hostname where it seems less relevant?

Reply:
Yep, my bad! The integer support was supposed to be on Port and not on Host